pom.xml

```diff
@@ -2023,6 +2023,46 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>com.github.ekryd.sortpom</groupId>
+                    <artifactId>sortpom-maven-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <executions>
+                        <execution>
+                            <id>sortpom-warn</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Warn</verifyFail>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>sortpom-fail</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Stop</verifyFail>
+                            </configuration>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <createBackupFile>false</createBackupFile>
+                        <keepBlankLines>true</keepBlankLines>
+                        <nrOfIndentSpace>4</nrOfIndentSpace>
+                        <sortOrderFile>${air.main.basedir}/.mvn/sortpom/sortorder.xml</sortOrderFile>
+                        <sortDependencyManagement>scope,groupId,artifactId</sortDependencyManagement>
```

Comment:
Remove, the default is `fallback` which will use `sortDependencies` (no need to duplicate)

Reply:
Applied